inspire_json_merger/utils.py

```diff
@@ -31,6 +31,8 @@
 
 split_on_re = re.compile(r'[\.\s-]')
 
+ORDER_KEY = "pos"
```

Comment:
```suggestion
ORDER_KEY = "__pos"
```

to make sure it won't ever conflict with actual keys.

Reply:
:+1: